native/cocos/bindings/auto/jsb_physics_auto.h

```diff
@@ -278,8 +277,8 @@ bool js_register_cc_physics_TerrainShape(se::Object *obj); // NOLINT
 SE_DECLARE_FUNC(js_physics_TerrainShape_getAABB);
 SE_DECLARE_FUNC(js_physics_TerrainShape_getBoundingSphere);
 SE_DECLARE_FUNC(js_physics_TerrainShape_getGroup);
-SE_DECLARE_FUNC(js_physics_TerrainShape_getImpl);
```

Comment:
Don't have to commit these changes as CI will generate binding codes automatically.